yaml/pois.yaml

```diff
@@ -172,6 +172,14 @@ filters:
     min_zoom: 17
     output:
       kind: {col: tags->zoo}
+  - filter: {waterway: waterfall}
```

Comment:
Considering Nevada Falls, one of the most important waterfalls in Yosemite NP is tagged `natural=waterfall`, we should include them. Good catch!